modules/console/pages/config/security/authorization.adoc

```diff
@@ -2,20 +2,37 @@
 :page-aliases: console:single-sign-on/authorization.adoc, manage:security/console/authorization.adoc, console:features/role-bindings.adoc, console:reference/role-bindings.adoc, reference:console/role-bindings.adoc
 
-Redpanda Console uses role-based access control (RBAC) to manage and restrict access to features in the user interface. RBAC allows you to control access to different resources in Redpanda Console by binding roles (admin, editor, viewer) to users. These role bindings determine the permissions that users have when interacting with Redpanda Console.
-
 [NOTE]
 ====
 include::shared:partial$enterprise-and-console.adoc[]
 ====
 
-RBAC behavior depends on your authentication configuration in Redpanda Console. When using static credentials, you must give users permissions to use Redpanda Console by defining RBAC rules in the Redpanda Console configuration file. If authentication for an API is configured to impersonate users, Redpanda Console RBAC rules are ignored for that API and permissions are derived only from the roles assigned in Redpanda. For more information, see the xref:console:config/security/authentication.adoc[].
+Redpanda Console supports two authorization modes, depending on whether *user impersonation* is enabled in the authentication configuration:
+
+[cols="1,2"]
+|===
+| Mode | Who Evaluates Permissions?
+
+| **Impersonation enabled** (`impersonateUser: true`)
+| Redpanda authorizes requests based on internal RBAC and ACLs.
+`roleBindings` in the Redpanda Console configuration are ignored.
+
+| **Impersonation disabled** (`impersonateUser: false`)
+| Redpanda Console enforces access using static `roleBindings` from the configuration file.
+Redpanda treats all requests as coming from a single service account.
+|===
+
+Even when using Redpanda Console roles, users must still have ACLs in Redpanda if they are interacting with Redpanda through the UI.
+
+For more information about authentication options, see the xref:console:config/security/authentication.adoc[authentication configuration].
 
 == Roles
 
-Roles defined in Redpanda Console are separate from roles in Redpanda. Roles in Redpanda Console control what a user can see and do within the Console interface and only apply to endpoints where user impersonation is disabled. For example, while an admin user in Redpanda Console can generate debug bundles, the user must also have the corresponding `MANAGE_DEBUG_BUNDLE` ACL in Redpanda to execute the operation.
+Roles in Redpanda Console determine the actions users can perform in the UI. These roles are defined independently from Redpanda roles and only apply when impersonation is disabled. When impersonation is enabled, user permissions are derived entirely from Redpanda's internal role and ACL configuration.
 
-Redpanda Console comes with the following roles:
+For example, even if a user is assigned the Admin role in Console, they must also have the appropriate Redpanda ACLs to successfully perform protected actions.
```

Comment:
exactly!, @mattschumpert, this piece was surprising when I found out. If I'm giving my user Console's built-in "Admin", why do I have to keep granting it more permissions? As a user, it makes no sense at all.